tensorflow/tools/docker/Dockerfile.gpu

```diff
@@ -1,9 +1,10 @@
-FROM nvidia/cuda:7.5-cudnn4-runtime
+FROM nvidia/cuda:7.5-cudnn4-devel
```

Comment:
This container should not depend on devel as it would make it a lot bigger. And this container is what we recommend people to use for runtime. It should be small.